src/components/panel/panel.js

```diff
@@ -1119,7 +1119,10 @@ MdPanelRef.prototype._createPanel = function() {
 
           if (self._config['disableParentScroll']) {
             self._restoreScroll = self._$mdUtil.disableScrollAround(
-                null, self._panelContainer);
+              null,
+              self._panelContainer,
+              { doNotCreateScrollMaskElement: true }
```

Comment:
I would rather call this
```js
disableScrollMask: true
```

It's possible to name it that way, because the function is actually called `disableScrollAround`.